consumers/campaigner/main.go

```diff
@@ -90,18 +90,18 @@ func (h *MessageHandler) HandleMessage(m *nsq.Message) (err error) {
 	if err != nil {
 		logEntry.WithError(err).Error("unable to prepare campaign template data")
 
-		err = setStatusFailed(ctx, h.s, campaign)
+		err = logFailedCampaign(ctx, h.s, campaign, "failed to prepare campaign: parseTemplate")
```

Comment:
i think its more descriptive if you just set the description to `failed to parse template`